autometa/taxonomy/get_lca_taxa.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+
+import os
+import glob
+import logging
+import pyreadr
+
+import pandas as pd
+import numpy as np
+
+from autometa.taxonomy.lca import LCA
+from autometa.taxonomy.ncbi import NCBI, NCBI_DIR
+
+logger = logging.getLogger(__name__)
+
+# TODO: Currently hardcoded. Need to change it. Not sure how.
+
+lca = LCA(dbdir="/media/bigdrive1/Databases/autometa_databases", verbose=True)
+ncbi = NCBI(dirpath="/media/bigdrive1/Databases/autometa_databases", verbose=True)
```

Comment:
Not sure why you have these hardcode here?

Why not instantiate them in `main(...)`?